ci/run_python_tests.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+set -euo pipefail
+
+TEST_CMD=$1
+TEST_ARGS=$2
+
+# Functions
+function fn_die() {
+  echo -e "$1" >&2
+  exit "${2:-1}"
+}
+
+echo "=== Checking if GITHUB_TOKEN is set ==="
+if [ -z "${ZEN_REPO_TOKEN:-}" ]; then
+  fn_die "ZEN_REPO_TOKEN variable is not set. Exiting ..."
+fi
+
+CURRENT_DIR=$PWD
+
+# Step 1
+echo "" && echo "=== Pull latest zen release ===" && echo ""
+
+json_data=$(curl -L   -H "Accept: application/vnd.github+json"   -H "Authorization: Bearer ${ZEN_REPO_TOKEN}"   -H "X-GitHub-Api-Version: 2022-11-28"   https://api.github.com/repos/HorizenOfficial/zen/releases)
+
+deb_url=""
+deb_asc_url=""
+deb_sha256_url=""
+deb_name=""
+deb_asc_name=""
+deb_sha256_name=""
+
+while IFS= read -r main_array_item; do
```

Comment:
Rodrigo is you are trying to get the latest zen release why to use while loop? It is already ordered in json_data

```
curl -sL -H "Accept: application/vnd.github+json" -H "Authorization: Bearer TOKEN" -H "X-GitHub-Api-Version: 2022-11-28" https://api.github.com/repos/HorizenOfficial/zen/releases | jq '.[0]'
```

Reply:
Because the first one could be a release candidate or bitcore, we want the first which is NOT an rc or bitcore

Reply:
implemented in the new version